src/LinuxTracingIntegrationTests/LinuxTracingIntegrationTest.cpp

```diff
@@ -337,7 +340,7 @@ class LinuxTracingIntegrationTestFixture {
   [[nodiscard]] orbit_grpc_protos::CaptureOptions BuildDefaultCaptureOptions() {
     orbit_grpc_protos::CaptureOptions capture_options;
     capture_options.set_trace_context_switches(true);
-    capture_options.set_pid(puppet_.GetChildPid());
+    capture_options.set_pid(puppet_.GetChildPidNative());
```

Comment:
Shouldn't capture options contains the non-native pid?

Reply:
Absolutely, thanks. Done.